src/main/java/pl/agh/edu/iisg/io/vmms/vmmsbackend/model/reservations/Reservation.java

```diff
@@ -0,0 +1,91 @@
+package pl.agh.edu.iisg.io.vmms.vmmsbackend.model.reservations;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.model.User;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.model.VMPool;
+
+import javax.persistence.*;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@Data
+@Entity
+@NoArgsConstructor
+@Table(name = "reservations")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
```

Comment:
Wydaje mi się że lepiej byłoby rozbić to na dwie tabele

Reply:
No właśnie zastanawialiśmy się nad tym i wyszło nam, że lepiej trzymać to w zunifikowany sposób na zasadzie reservation i reservation_details na pojedyncze zajęcia